.github/workflows/go.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         go-version-file: "go.mod"
 
     - name: Run Unit Tests
-      run: go test -v ./... -coverpkg=./... -coverprofile=coverage.txt
```

Comment:
Dropping `-v` prints only relevant logs, leading to a succinct summary of passing packages, with full logs only for failing tests.